app/src/main/java/org/openimis/imispolicies/CummulativeIndicators.java

```diff
@@ -154,6 +155,7 @@ public void run() {
 
     public void getCommulativeIndicators(String DateFrom, String DateTo){
         try {
+            ca.CheckInternetAvailable();
```

Comment:
If there is no connectivity, it will not stop the function.

Reply:
Fixed